components/cd-card/cd-card.html

```diff
@@ -0,0 +1,156 @@
+<link rel=stylesheet href="../cd-base.css" type="text/css" media=screen>
+<link rel=stylesheet href="../cd-grid/cd-grid.css" type="text/css" media=screen>
+<link rel=stylesheet href="../cd-flow/cd-flow.css" type="text/css" media=screen>
+<link rel=stylesheet href="../cd-utilities/cd-utilities.css" type="text/css" media=screen>
+<link rel=stylesheet href="../cd-typography/cd-typography.css" type="text/css" media=screen>
+<link rel=stylesheet href="../cd-button/cd-button.css" type="text/css" media=screen>
+<link rel=stylesheet href="../cd-read-more/cd-read-more.css" type="text/css" media=screen>
+<link rel=stylesheet href="cd-card.css" type="text/css" media=screen>
+
+<div class="cd-layout-container">
+  <div class="cd-container cd-grid cd-grid-3-col">
+
+    <article role="article" class="cd-card node--view-mode-card [ cd-flow ]">
```

Comment:
I see five instances, but highlighting one: `.node--view-mode-card` seems to be a drupal-specific class and it isn't referenced in the CSS.